app/command_msg.go

```diff
@@ -9,6 +9,9 @@ import (
 	goi18n "github.com/mattermost/go-i18n/i18n"
 	"github.com/mattermost/mattermost-server/v5/mlog"
 	"github.com/mattermost/mattermost-server/v5/model"
+	"github.com/mattermost/mattermost-server/v5/store"
+
+	"github.com/pkg/errors"
```

Comment:
We can use the standard library for this.

Reply:
Yep, this was a mistake. I use goimports on save so I was added by him, even I didn't know that `github.com/pkg/errors` had the same methods `as` and `is`